sdk/src/androidTest/java/ly/count/android/sdk/ModuleRemoteConfigTests.java

```diff
@@ -88,12 +88,14 @@ public void automaticRCTriggers() {
             }
 
             countly.deviceId().changeWithMerge("dd");
-            Assert.assertEquals(intendedCount, triggerCounter[0]);//changing device ID with merging should not create a request
+            Assert.assertEquals(++intendedCount, triggerCounter[0]);//changing device ID with merging should create a request
 
             countly.deviceId().changeWithoutMerge("dd11");
-            //todo the current behaviour is slightly out of spec as it would download RC after the RQ would have executed that request
-            //todo this should be updated once the RQ is reworked
-            Assert.assertEquals(intendedCount, triggerCounter[0]);//changing device ID without merging should create a request
+            if (a == 0) {
+                Assert.assertEquals(intendedCount, triggerCounter[0]);//changing device ID without merging won't create a request because all consents are revoked
+            } else {
+                Assert.assertEquals(++intendedCount, triggerCounter[0]);//changing device ID without merging would create a request because consent is not required
```

Comment:
with?

Reply:
they are both without, it is just that that test case is written in a for loop to test multiple scenarios. So I did not break that behavious